high_card.rb

```diff
@@ -2,4 +2,24 @@
 ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
-players = []
+# players = []
+
+# heartsCards = []
+# spadesCards = []
+# clubsCards = []
+# diamondsCards = []
+
+
+# while i < suits.length
+cardDeck = suits.each do |i|
+    ranks.each do |j|
+      cardDeck.push([suit[i], rank[j]])
+    end
+  end
+# end
```

Comment:
Good start here. Nice use of nested enumerables to generate a deck of cards. Next step would be to `.shuffle` them. Thankfully Ruby provides us with a very easy way of doing that...